Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghost:6.5.3-alpine
+
+# Add the Object Store storage adapter
+ADD https://github.com/CodeForAfrica/ghost-object-store-storage-adapter.git content/adapters/storage/object-store
```

Comment:
Would [using `COPY`](https://www.docker.com/blog/docker-best-practices-understanding-the-differences-between-add-and-copy-instructions-in-dockerfiles/) or a verified git fetch be a better approach here?
Prefer fetching with git pinned to a tag or branch to make this reproducable.